docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -40,48 +33,41 @@ For an up-to-date documentation, run `ng help @o3r/components:extractor`
       "@your/o3r-components"
     ]
   }
-},
+}
 ```
 
-__Note:__ This options will not search for the duplicate configurations in libraries.
+__Note:__ This option will not search for the duplicate configurations in libraries.
 
 The __strict mode__ option should be used for production builds, it will throw an error if something not supported by the
-cms is found in the config. You can set it to `false` to allow the generation of metadata including the unknown types to
+CMS is found in the config. You can set it to `false` to allow the generation of metadata including the unknown types to
 ease issues fixing, and with the errors logged as warnings.
 
-Here is an example on a library
-
+Here is an example on a library:
 ```json
-    {
-  //...
-  "extract-configurations": {
-    "builder": "@o3r/components:extractor",
-    "options": {
-      "tsConfig": "modules/@scope/area/components/tsconfig.metadata.json",
-      "configOutputFile": "modules/@scope/area/components/dist/component.config.metadata.json",
-      "componentOutputFile": "modules/@scope/area/components/dist/component.class.metadata.json",
-      "strictMode": true
-    }
+"extract-components": {
+  "builder": "@o3r/components:extractor",
+  "options": {
+    "tsConfig": "modules/@scope/area/components/tsconfig.metadata.json",
+    "configOutputFile": "modules/@scope/area/components/dist/component.config.metadata.json",
+    "componentOutputFile": "modules/@scope/area/components/dist/component.class.metadata.json",
+    "strictMode": true
   }
-  //...
 }
 ```
 
-In case you have a lib in a mono repository, it's important to specify both configOutputFile and componentOutputFile, by
-default if not specified, it will be generated at the root of the project
+In case you have a library in a mono repository, it's important to specify both configOutputFile and componentOutputFile. By default, if it is not specified, it will be generated at the root of the project.
 
-For the tsconfig, you should include only the files that you want to parse, i.e. components, modules and config. You can
+For the tsconfig, you should include only the files that you want to parse, i.e. components and config. You can
 extend your tsconfig, and just override the fields that you need.
 
 Example of a tsconfig :
 
 ```json
-    {
+{
   "extends": "./tsconfig",
   "rootDir": ".",
   "include": [
     "src/**/*.component.ts",
-    "src/**/*.module.ts",
     "src/**/*.config.ts"
   ],
```

Comment:
In the component file example maybe you should mention the flavor your did with angular signals as an alternative now